x-pack/plugins/endpoint/public/embeddables/resolver/store/middleware.ts

```diff
@@ -30,6 +31,26 @@ function flattenEvents(children: ResolverNode[], events: ResolverEvent[] = []):
   }, events);
 }
 
+type RelatedEventAPIResponse = Error | { events: ResolverEvent[] };
+async function* getEachRelatedEventsResult(
+  eventsToFetch: ResolverEvent[],
+  httpGetter: HttpHandler
+): AsyncGenerator<[ResolverEvent, RelatedEventAPIResponse], any, any> {
```

Comment:
don't think you need these explicit any's

Reply:
I took them off and `tsc` doesn't complain: https://github.com/elastic/kibana/pull/64121/commits/e2d87f4571f1dbf44004f0b361e120609f03d548 Thanks @dplumlee